### ISSUE TITLE
i can't install 2017.4 ! on ubuntu 16.04. I follow the process and it install 2016.10

### ISSUE THREAD

**Issue by innovate-data** [OPM/opm-simulators#1172]

**Comment by andlaus** [comment]
could you please be a bit more specific?

**Comment by andlaus** [comment]
e.g.: are you using WSL? if yes, you need the creator's update for OPM 2017.04

**Comment by innovate-data** [comment]
yes i am using WSL what should i do now. what is creator's update ??

**Comment by andlaus** [comment]
it's the latest version of windows: https://support.microsoft.com/en-us/instantanswers/d4efb316-79f0-1aa1-9ef3-dcada78f3fa0/get-the-windows-10-creators-update

it is currently in the rollout phase, but Microsoft is not yet updating everyone and enterprise IT departments  will probably lack behind significantly if the usual patten holds.

**Comment by blattms** [comment]
There are newer versions of opm for Ubuntu and there are reports that they work. Closing
